.github/workflows/check_labels.yml

```diff
@@ -0,0 +1,47 @@
+name: Check Labels
+
+# Every PR should have a label and some labels should include an update to the release notes
+on:
+  pull_request:
+    branches-ignore:
+      - 'main'
+    types: [ synchronize, labeled, unlabeled ]
```

Comment:
Is `synchronize` not going to cause similar issues as before?

Reply:
Only if we push and label at the same time, but in that case unlabeled could cause the same issue. It's a tough one to fully fix as long as github does not support this themselves but I guess it's the closest we can get.

Reply:
Ah okay that should be fine. I didn't know when `synchronize` is triggered